index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
Well, the PR says `usually represents distance` similar to several other spots in the current spec which say usually. I don't think we can say anything definitive because I don't believe anything says manufacturers have to tie it to anything at all, it can be completely arbitrary.

From the [Android API docs](https://developer.android.com/reference/android/hardware/camera2/CameraCharacteristics.html#LENS_INFO_FOCUS_DISTANCE_CALIBRATION):

> APPROXIMATE and CALIBRATED devices report the focus metadata in units of diopters (1/meter), so 0.0f represents focusing at infinity, and increasing positive numbers represent focusing closer and closer to the camera device. The focus distance control also uses diopters on these devices.
> **UNCALIBRATED devices do not use units that are directly comparable to any real physical measurement**, but 0.0f still represents farthest focus, and android.lens.info.minimumFocusDistance represents the nearest focus the device can achieve.

The Windows API clearly states millimeters. So, given the 3 different sources of info, Android says an uncalibrated device will use arbitrary units, a calibrated device will use diopters, and on Windows its millimeters (maybe).